src/modules/common/data/tutorial-educators.js

```diff
@@ -0,0 +1,29 @@
+export default {
+  pages: [
+    {
+      title: 'Welcome to the WildCam Lab!',
+      image: require('../../../images/tutorial/educator-tutorial-1.gif'),
+      text: 'The WildCam Lab is a tool that allows your students to use WildCam images and data for inquiry-based ecology projects. Using the lab, you can manage classrooms, explore and download data, and assign photo sets for your students to identify.',
+    },
+    {
+      title: 'Set up a classroom',
+      image: require('../../../images/tutorial/educator-tutorial-2.gif'),
+      text: 'The first time you set up a classroom, register by providing information about yourself. You only have to do this once. Then, create a new classroom and invite your students by sending them a link. From here, you can monitor how many classifications your students make on WildCam and you can create assignments.',
+    },
+    {
+      title: 'Create an assignment',
+      image: require('../../../images/tutorial/educator-tutorial-3.gif'),
+      text: 'An assignment is a set of WildCam images that you have selected for your students to identify and then download their own data. Within a classroom, click the assignments tab and create a new assignment. Select the number of images you want each student to identify, and select which students in your class will receive this assignment. Next, select the set of images your students will identify by filtering the map and clicking "Select for Assignment."',
+    },
+    {
+      title: 'Explore and download data',
+      image: require('../../../images/tutorial/explorer-tutorial-2.gif'),
+      text: 'The Data page allows you and your students to explore and download data from images that the public has identified on WildCam Gorongosa. Click the "View Filters" button and select to items to filter. Click the download button to download a spreadsheet of the data and share it with your students. Click on an orange circle to see a preview of images from that camera that match your filters.',
```

Comment:
Same typo: "select to items to filter" should probably be "select the items to filter".